TOOLS/dylib_unhell.py

```diff
@@ -41,83 +41,106 @@ def otool(objfile, rapths):
-    pathRe = re.compile(r"^\s*path (.*) \(offset \d*\)$")
+    command = f"otool -l '{objfile}' | grep -A2 LC_RPATH | grep path"
+    path_re = re.compile(r"^\s*path (.*) \(offset \d*\)$")
 
     try:
-        result = subprocess.check_output(command, shell = True, universal_newlines=True)
-    except:
+        result = subprocess.check_output(command, shell=True, universal_newlines=True)
+    except Exception:
         return rpaths
 
     for line in result.splitlines():
-        line_clean = pathRe.search(line).group(1).strip()
+        line_clean = path_re.search(line).group(1).strip()
         # resolve @loader_path
-        if line_clean.startswith('@loader_path/'):
-            line_clean = line_clean[len('@loader_path/'):]
-            line_clean = os.path.normpath(os.path.join(os.path.dirname(objfile), line_clean))
+        if line_clean.startswith("@loader_path/"):
+            line_clean = line_clean[len("@loader_path/"):]
+            line_clean = os.path.join(os.path.dirname(objfile), line_clean)
+            line_clean = os.path.normpath(line_clean)
         rpaths.append(line_clean)
 
     return rpaths
 
 def get_rpaths_dev_tools(binary):
-    command = "otool -l '%s' | grep -A2 LC_RPATH | grep path | grep \"Xcode\\|CommandLineTools\"" % binary
+    command = (
+        f"otool -l '{binary}' | grep -A2 LC_RPATH | grep path | "
+        'grep "Xcode\\|CommandLineTools"'
+    )
     result  = subprocess.check_output(command, shell = True, universal_newlines=True)
-    pathRe = re.compile(r"^\s*path (.*) \(offset \d*\)$")
-    output = []
-
-    for line in result.splitlines():
-        output.append(pathRe.search(line).group(1).strip())
-
-    return output
+    path_re = re.compile(r"^\s*path (.*) \(offset \d*\)$")
+    return [
+        path_re.search(line).group(1).strip()
+        for line in result.splitlines()
+    ]
 
 def resolve_lib_path(objfile, lib, rapths):
     if os.path.exists(lib):
         return lib
 
-    if lib.startswith('@rpath/'):
-        lib = lib[len('@rpath/'):]
+    if lib.startswith("@rpath/"):
+        lib = lib[len("@rpath/"):]
         for rpath in rapths:
             lib_path = os.path.join(rpath, lib)
             if os.path.exists(lib_path):
                 return lib_path
-    elif lib.startswith('@loader_path/'):
-        lib = lib[len('@loader_path/'):]
+    elif lib.startswith("@loader_path/"):
+        lib = lib[len("@loader_path/"):]
         lib_path = os.path.normpath(os.path.join(objfile, lib))
         if os.path.exists(lib_path):
             return lib_path
 
-    raise Exception('Could not resolve library: ' + lib)
+    raise Exception("Could not resolve library: " + lib)
 
 def check_vulkan_max_version(version):
     try:
-        result = subprocess.check_output("pkg-config vulkan --max-version=" + version, shell = True)
+        cmd = f"pkg-config vulkan --max-version={version}"
+        subprocess.check_output(cmd, shell=True)
```

Comment:
wonder what the reason for this inline > variable (cmd) change is?

in the other cases they stayed as they were. if we change those, it should be consistent.

Reply:
The goal was just to make the line shorter. I've switched that to a multi-line function call instead to keep it consistent.